pkg/utils/utils.go

```diff
@@ -43,3 +44,11 @@ func HashCode(msg string) string {
 	h.Write([]byte(msg))
 	return string(h.Sum(nil))
 }
+
+func HashedMatch(namespace string, match string) string {
```

Comment:
I suggested to rename:
- `HashedMatch` to `generateNamespacedMatchExpr`
- `HashedMatchRegex` to `generateNamespacedMatchRegExpr`